source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+- We aim to be able to restore all service within 48 hours of disruption.
+
+## Disaster Scenarios
+
+| Disaster                                                                                                           | Severity | Likelihood | Recovery/Mitigation                                                                                                                                                                                                                       |
+|--------------------------------------------------------------------------------------------------------------------|----------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Disruption to hosting services                                                                                     | High     | Low        | - Following recovery of the platform, we may need to [re-deploy our applications](#issues-at-cloud-platform-level) |
+| AWS Region outage                                                                                                  | High     | Low        | - Wait<br>- This may lead to Disruption to hosting services (above) |
+| Loss of data due to misconfiguration of AWS resources in cloud-platform-environments repo                          | Medium   | Low        | - [Restore database from RDS snapshot](#database-recovery) |
+| Intentional disruption by malicious actor e.g. Distributed Denial of Service (DDoS)                                | High     | Medium     | - [Horizontal auto-scaling] with sensible scaling ceiling<br>- Rate limiting on NGINX ingress<br>- [AWS Shield Standard] (managed DDoS protection service) safeguards applications running on AWS. CP are looking into rolling out [AWS Shield Advanced] for more comprehensive integrated protection |
+| Deletion / corruption of application database data                                                                 | Medium   | Low        | - [Restore from snapshot](#database-recovery) |
+| Dependency poisoning attacks                                                                                       | High     | Low        | - Using Dependabot<br>- Scanning on pipeline - NPM audit<br>- Container scanning<br>Dependency assessment criteria<br>Guardduty monitors VPC flow logs for unusual activity |
+| Accidental misconfiguration of code leading to data loss                                                           | High     | Low        | - Manually fix in the DB where possible<br>- DB snapshot	restore |
+| Entra ID disruption (loss of authentication function)                                                              | High     | Low        | - Wait |
+
```

Comment:
Do we want to mention what happens if github has an outage? In addition to blocking our deploy pipeline, this would affect scheduled metadata ingestions.